jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-aws.yaml

```diff
@@ -19,9 +19,10 @@
         - email-ext:
             recipients: '{emails}'
         - gcs-uploader
+    # Need a minimum of 8 kube-system pods ready before declaring cluster ready
```

Comment:
These are the set of essential kube-system pods necessary for E2E test suites to succeed. I will condense this list and reword all the comments.

```
etcd-server-kubernetes-master               
kube-apiserver-kubernetes-master            
kube-controller-manager-kubernetes-master   
kube-dns-v15-gandm                          
kube-scheduler-kubernetes-master            
l7-default-backend-v1.0-66izg               
l7-lb-controller-v0.6.3-kubernetes-master
kube-addon-manager-kubernetes-master        
```